docs/devguide/contributing.rst

```diff
@@ -35,3 +35,76 @@ negative votes, the team will grant commit privileges.
 The core team will also be responsible for removing commit privileges when
 appropriate - for example for malicious merge behavior or just inactivity over
 an extended period of time.
+
+
+Quality gates for the pull requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are three quality gates to ensure the pull requests quality, `Hound`_ for
```

Comment:
I think you miss the coveralls for unit-testing coverage report, the coverage should not decrease a lot for a new PR (though the specific value is still TBD and may varies on different cases)